helpers/l10n.ts

```diff
@@ -39,7 +39,7 @@ export function translateWithParameters(
   if (message) {
     return parameters
       .map((parameter) => String(parameter))
-      .reduce((acc, parameter, index) => acc.replace(`{${index}}`, parameter), message);
+      .reduce((acc, parameter, index) => acc.replace(`{${index}}`, () => parameter), message);
```

Comment:
You actually don't need the `` `${parameter}` `` like I do because you have the `.map((parameter) => String(parameter))` just before the reduce, and I dropped it on my PR.